charts/hush-sensor/values.yaml

```diff
@@ -277,6 +277,13 @@ sentry:
     # Custom annotations for sentry service account
     annotations: {}
 
+  # Credentials for sentry secret store crawler
+  # aws:
+  #   irsa: arn of role to configure with irsa annotations
```

Comment:
The creds are not for the crawler, they are for sentry.
So IMO we shouldn't mention crawler in the description.

If in future we will have another crawler the user will have to augment the same role with more permissions. So it will be a role for many "crawlers".